README.md

```diff
@@ -241,10 +241,10 @@ The way the vagrant box is created is by making a network with a static IP on th
 This means that you cannot connect to it from some other box.
 
 ```bash
-# Attach to the endpoint (self-signed certs in dev mode requires skipping validation)
-# cf-dev.io simply resolves to the static IP 192.168.77.77 that vagrant provisions
-# This DNS resolution may fail on certain DNS providers that block resolution to 192.168.*
-# Unless you've changed the default credentials in the configuration it's admin/changeme
+# Attach to the endpoint (self-signed certs in dev mode requires skipping validation).
+# cf-dev.io resolves to the static IP that vagrant provisions.
+# This DNS resolution may fail on certain DNS providers that block resolution to 192.168.0.0/16.
+# Unless you've changed the default credentials in the configuration, it is admin/changeme.
```

Comment:
Minor thing: is the `it's` change to reduce use of contractions since this is a somewhat formal document? Curious because `you've` is on the same line

Reply:
Indeed it is! My poor eyes caught this one, but fooled me with `you've`.